lineups/endoftheroad_2022.txt

```diff
@@ -0,0 +1,105 @@
+Pixies
+Fleet Foxes
+Bright Eyes
+Khruangbin
+The Magnetic Fields
+Aldous Harding
+Kurt Vile
+Perfume Genius
+Kevin Morby
+Tinariwen
+black midi
+Lucy Dacus
+Nilüfer Yanya
+Soccer Mommy
+The Weather Station
+Sudan Archives
+Hurray For The Riff Raff
+Nala Sinephro
+Porridge Radio
+Yard Act
+Battles
+Ross From Friends
+Moor Mother
+Durand Jones & The Indications
+Anaïs Mitchell
+Ryley Walker
+Gabriels
+Emma-Jean Thackray
+Steam Down
+Priya Ragu
+BCUC
+Cassandra Jenkins
+Dehd
+Purling Hiss
+Los Bitchos
+NewDad
+Les Filles De Illighadad
+Wu-Lu
+SCALPING
+I. JORDAN
+Kareem Ali
+Circuit Des Yeux
+Ural Thomas & The Pain
+Mike Polizze
+Yasmin Williams
+Jake Xerxes Fussell
+Skullcrusher
+The Lounge Society
+Fruit Bats
+The Golden Dregs
+Christian Lee Hutson
+Xenia Rubinos
+Alabaster dePlume
+Jana Horn
+Ahmed Fakroun
+Part Chimp
+Sinead O’Brien
+Grace Cummings
+audiobooks
+Margo Cilker
+Mandy, Indiana
```

Comment:
retroactive FYI: artists can't have commas in their name. The cache script splits days by looking for commas, so this festival would then have a day 0 for all artists and a day 'Indiana' for an artist called 'Mandy'.

If you feel like making the parser a bit better, it could check for failure on trying to parse whatever is after the comma as a number. If it's not, we mistakenly left a comma in an artist and we should either blow up and fail (easiest) or ignore any day logic and treat the whole thing as the artist.

Zero pressure to add it, it's easy to catch any missed commas by looking at the filter page after you add the festival and seeing some weird days in that list instead of no days. I just got lazy. Figured I'd mention it if you're keen on more to work on, feel free to continue down your own feature req list

Reply:
This is weird. When I ran locally to test the lineup file I thought it correctly picked everything up. Not sure what happened there but I will re-run it locally and ensure I'm checking in the right place.

Re the parser, yes I can think of a few ways to try and catch errors. Probably the easiest will be to have a rule that whatever the format of the first (non-empty) line is (i.e. - comma with number, or no comma and number) is then expected to be the format for all subsequent lines, and any deviation results in an error and abort. I'll have a look at proposing something

Reply:
Yeah it picks everything up fine with no errors, it just shows up when you build it and navigate to the customize lineup page and look at the days filter. The readme update  I added around testing should help catch these